localstack/services/apigateway/context.py

```diff
@@ -154,7 +154,7 @@ def is_v1(self):
     def cookies(self):
```

Comment:
nit: was this change required to get the velocity functionality working? While we're at it, maybe we can also add a type hint, to make it more explicit:
```
    def cookies(self) -> Optional[List[str]]:
```

Reply:
Not required, but it's not correct as it stands. This was just an opportunity to fix it.